src/Illuminate/Console/ConfirmableTrait.php

```diff
@@ -13,8 +20,12 @@ trait ConfirmableTrait
      * @param  \Closure|bool|null  $callback
      * @return bool
      */
-    public function confirmToProceed($warning = 'Application In Production!', $callback = null)
+    public function confirmToProceed($warning, $callback = null)
     {
+        if ($warning === null){
```

Comment:
if warning can be `null` you must update the phpdoc and add a default param.

Reply:
The original intention was to have `"Application In {$this->environmentToConfirm}!"`, but you're not allowed expressions in the default.